apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx

```diff
@@ -0,0 +1,388 @@
+          accusedAddress={workingCase.accusedAddress}
+          defender={{
+            name: workingCase.defenderName || '',
+            email: workingCase.defenderEmail,
+            phoneNumber: workingCase.defenderPhoneNumber,
+          }}
+        />
+      </Box>
+      {(workingCase.accusedAppealDecision === CaseAppealDecision.POSTPONE ||
+        workingCase.prosecutorAppealDecision === CaseAppealDecision.POSTPONE) &&
+        workingCase.rulingDate &&
+        (user?.role === UserRole.JUDGE ||
+          user?.role === UserRole.REGISTRAR) && (
+          <Box marginBottom={7}>
+            <AppealSection
+              rulingDate={workingCase.rulingDate}
+              accusedGender={
+                // Handle missing gender
+                workingCase.accusedGender ?? CaseGender.OTHER
+              }
+              accusedAppealDecision={workingCase.accusedAppealDecision}
+              prosecutorAppealDecision={workingCase.prosecutorAppealDecision}
+              accusedPostponedAppealDate={
+                workingCase.accusedPostponedAppealDate
+              }
+              prosecutorPostponedAppealDate={
+                workingCase.prosecutorPostponedAppealDate
+              }
+              setAccusedAppealDate={setAccusedAppealDate}
+              setProsecutorAppealDate={setProsecutorAppealDate}
+              withdrawAccusedAppealDate={withdrawAccusedAppealDate}
+              withdrawProsecutorAppealDate={withdrawProsecutorAppealDate}
+              isAppealDeadlineExpired={
+                workingCase.isAppealDeadlineExpired ?? false
+              }
+              isAppealGracePeriodExpired={
+                workingCase.isAppealGracePeriodExpired ?? false
+              }
+            />
+          </Box>
+        )}
+      <Box marginBottom={5}>
+        <Accordion>
+          <PoliceRequestAccordionItem workingCase={workingCase} />
+          <CourtRecordAccordionItem workingCase={workingCase} />
+          <RulingAccordionItem workingCase={workingCase} />
+          <AccordionItem
+            id="id_4"
+            label={`Rannsóknargögn (${
+              workingCase.files ? workingCase.files.length : 0
+            })`}
+            labelVariant="h3"
+          >
+            <CaseFileList
+              caseId={workingCase.id}
+              files={workingCase.files || []}
+              canOpenFiles={canCaseFilesBeOpened()}
+            />
+          </AccordionItem>
+        </Accordion>
+      </Box>
+      <Box marginBottom={user?.role === UserRole.PROSECUTOR ? 7 : 15}>
+        <Box marginBottom={3}>
+          <PdfButton
+            caseId={workingCase.id}
+            title="Opna PDF kröfu"
+            pdfType="request"
+          />
+        </Box>
+        <PdfButton
+          caseId={workingCase.id}
+          title="Opna PDF þingbók og úrskurð"
+          pdfType="ruling"
+        />
+      </Box>
+      {user?.role === UserRole.PROSECUTOR && (
```

Comment:
This section should only be visible to prosecutors belonging to the owning prosecutors office.

Reply:
Thank you 🙏  Maybe we should add error handling in the backend for this as well? Return 403?

Reply:
I already did 👍